lib/question.js

```diff
@@ -1007,9 +1013,10 @@ module.exports = {
                     return;
                 }
 
-                async.eachSeries(_.range(count), (iter, callback) => {
-                    job.verbose(`Test ${iter + 1}`);
-                    this._runTest(job, showDetails, question, course_instance, course, authn_user_id, (err, ret_success) => {
+                async.eachSeries(_.range(count * test_types.length), (iter, callback) => {
+                    let type = test_types[iter % test_types.length];
+                    job.verbose(`Test ${iter + 1}, type ${type}`);
```

Comment:
```suggestion
                    job.verbose(`Test ${Math.floor(iter/test_types.length) + 1}, type ${type}`);
```

Very minor, but this makes it so clicking "Test 100 times" actually counts up to 100, not 300.